test/smoke_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -x
+set -e
+set -a
+
+NODE_MODE=$1 #single or split mode
+
+COMPOSE_CMD="NODE_IMAGE=$IMAGE docker-compose -p $NODE_MODE -f test/docker-compose.$NODE_MODE.yaml"
+
+# Up compose
+/bin/bash -c "$COMPOSE_CMD up -d --wait --quiet-pull"
+
+# set tests variables and run tests
+GITHUB_VARS=$(env | awk -F '=' -v ORS==" " '/^GITHUB_/ {print "-e " $1 "=" $2}')
+RAND_NUM=$(echo $RANDOM$RANDOM$RANDOM | cut -c 1-10)
+
+RUN_TESTS="pytest pytest"
+if [ "$ALLURE_UPLOAD_REPORT" = "true" ]; then
+  RUN_TESTS="pytest allurectl watch --job-uid $RAND_NUM -- pytest"
+fi
+
+NODE_UUID=$(/bin/bash -c "$COMPOSE_CMD exec node cat /opt/wallarm/etc/wallarm/node.yaml | grep uuid | awk '{print \$2}'")
```

Comment:
If NODE_UUID is empty we need to fail in the same way as we do in Ingress
https://github.com/wallarm/ingress/blob/main/test/smoke/run-smoke-suite.sh#L160C7-L160C39